### DISCUSSION TITLE
PWA

### DISCUSSION THREAD

**Discussion by wisplite** [DustinBrett/daedalOS#49] [Feature Requests]
This is not a bug, just a feature request. It would be cool if you could add the site to your home screen as a PWA app, so you could get rid of the browser bar, which especially helps if you’re like me and using this on iOS. I don’t know how hard this would be to implement, but if possible, it would be cool to have.

**Comment by DustinBrett** [comment]
Thanks for the idea! I also would like to do this. I will add it to my list.

**Comment by xmha97** [comment]
+1, I want to add (install) this site (app) on my phone home screen to use full-screen mode.
